lib/apollo_upload_server/graphql_data_builder.rb

```diff
@@ -65,7 +65,7 @@ def verify_array_index!(path, index, size)
     def safe_json_parse(data)
       JSON.parse(data)
     rescue JSON::ParserError
-      nil
+      raise JSON::ParserError, "Malformed JSON in multipart form data"
```

Comment:
This function is no longer safe, so let's remove it and use JSON.parse directly where needed.